Microsoft.DotNet.Interactive.Jupyter/JupyterKernelHelper.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.DotNet.Interactive.Events;
+
+namespace Microsoft.DotNet.Interactive.Jupyter
+{
+    public static class Kernel
+    {
+        public static string input(
```

Comment:
we might need to have a password() gesture so that we can return a PasswordString object and make sure that the formatter for that and ToString() of it gets our encoded or not recorded. In the screenshot you print het content variable and the obfuscation is lost

Reply:
Updated to only cover the non-password input request in this PR. Make `InputRequested` even for non-password input and `PasswordRequested` event for password input. #155 was opened to track the work to add a gesture for get password input.